Tests/test_GAMutation.py

```diff
@@ -62,8 +62,8 @@ def _always_mutate(self, mutator, expected_percent):
                 num_mutations += 1
 
         percent_mutants = float(num_mutations) / float(self.num_trials)
-        assert percent_mutants > expected_percent, \
-               "Did not receive an acceptable number of mutations."
+        self.assertGreater(percent_mutants, expected_percent,
+                           "Did not receive an acceptable number of mutations.")
```

Comment:
Good. There maybe more cases like this where now that we've dropped Python 2.6 support we can use the more specific assert methods.